benchmark/fs/read-stream-throughput-dur.js

```diff
@@ -0,0 +1,50 @@
+// test the throughput of the fs.ReadStream class.
```

Comment:
Yeah, I just copied from the other benchmark. TBH I don't think the comment even needs to exist.